src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -301,6 +301,14 @@ public override Task<InitializeViewResult> InitializeObjectView(InitializeViewRe
                                     // Files tab is only supported in SQL Server, but files exists for all servers and used in detach database, cannot depend on files property to check the supportability
                                     ((DatabaseInfo)databaseViewInfo.ObjectInfo).IsFilesTabSupported = true;
                                     ((DatabaseInfo)databaseViewInfo.ObjectInfo).Filegroups = GetFileGroups(smoDatabase, databaseViewInfo);
+                                    try
+                                    {
+                                        // Cannot retrieve ServerFilestreamAccessLevel property for localdb instance as filestream level comes from the registry
```

Comment:
So if I'm understanding this correctly - what's happening here is that you're updating the dialog to specifically check if filestream is supported for enabling/disabling the table. But SMO uses the registry to populate the ServerFilestreamAccessLevel property - which means it will fail for anything that doesn't have access to that (which would include other platforms, including docker, I would expect).

So the try/catch here is just making it so we don't fail to open the dialog in those scenarios.

If the above is accurate then I have two suggestions: 

1. Update the comment to clearly indicate that - `ServerFilestreamAccessLevel uses the registry so will throw if that isn't available. In that case we just default to Disabled to ensure the UI reflects that`
2. Specifically set the property to disabled on a catch (but still log the error, although I would log it as an info and include a comment that this may be expected). Leaving it as undefined isn't the worst either - but I see no reason not to specifically just disable it in this scenario since we know it isn't available

@erinstellato-ms @shueybubbles This seem reasonable to you?

Reply:
@Charles-Gagnon making sure I understand...  there are scenarios where we will not be able to definitively confirm if the filestream option should be enabled or not, so in the case where we cannot, we will disable it, but include an info message.  Correct?  If so, then yes, seems reasonable.

Reply:
I didn't say "SMO uses the registry". SMO uses TSQL to invoke xp_reg_read on the server. Apparently `localdb` doesn't like xp_reg_read.
The question is whether ADS is trying to support localdb or not.

Reply:
@Charles-Gagnon , you are completely right. I can update comment and also do the #2. Thanks.

waiting for @shueybubbles and @erinstellato-ms comment on this one!!

Reply:
@shueybubbles Good point - we can update the text to be `ServerFilestreamAccessLevel uses xp_instance_regread so will throw if that isn't available. In that case we just default to Disabled to ensure the UI reflects that`

@erinstellato-ms Yup. Although I haven't investigated why localdb would throw in this case, since it should be able to access the registry...might just be an intended limitation of localdb itself.